website/docs/r/policy_host_transport_node.html.markdown

```diff
@@ -0,0 +1,164 @@
+---
+subcategory: "Fabric"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_host_transport_node"
+description: A resource to configure a HostTransportNode.
+---
+
+# nsxt_policy_host_transport_node
+
+This resource provides a method for the management of a HostTransportNode.
+This resource is supported with NSX 4.1.0 onwards.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_host_transport_node" "test" {
+  description       = "Terraform-deployed host transport node"
+  display_name      = "tf_host_transport_node"
+  site_path         = "/infra/sites/default"
+  enforcement_point = "default"
+
+  node_deployment_info {
+    ip_addresses = ["10.168.186.150"]
+
+    os_type    = "ESXI"
+    os_version = "7.0.3"
+
+    host_credential {
+      username   = "user1"
+      password   = "password1"
+      thumbprint = "thumbprint1"
+    }
+  }
+
+  standard_host_switch {
+    host_switch_mode    = "STANDARD"
+    host_switch_type    = "NVDS"
+    host_switch_profile = ["/infra/host-switch-profiles/2812c1e9-00e6-471a-9678-873c101d8f85"]
+
+    ip_assignment {
+      assigned_by_dhcp = true
+    }
+
+    transport_zone_endpoint {
+      transport_zone = "/infra/sites/default/enforcement-points/default/transport-zones/1b3a2f36-bfd1-443e-a0f6-4de01abc963e"
```

Comment:
Do we want to provide dependency here instead?

Reply:
Sounds good. Changed.